ksmt-test/build.gradle.kts

```diff
@@ -133,6 +138,27 @@ task<TestReport>("mergeTestReports") {
     }
 }
 
+task("mergeCSVFiles") {
```

Comment:
This csv merge task is a part of symfpu benchmarks and should be removed from there. Maybe, we can create a main function inside symfpu-benchmarks.